util/chunk/chunk.go

```diff
@@ -435,10 +410,21 @@ func (c *column) appendBytes(b []byte) {
 	c.finishAppendVar()
 }
 
-func (c *column) appendInterface(o interface{}) {
-	c.ifaces = append(c.ifaces, o)
-	c.appendNullBitmap(true)
-	c.length++
+func (c *column) appendTime(t types.Time) {
+	writeTime(c.elemBuf, t)
+	c.finishAppendFixed()
+}
+
+func writeTime(buf []byte, t types.Time) {
+	binary.BigEndian.PutUint16(buf, uint16(t.Time.Year()))
```

Comment:
Some field may excess the range of uint8

Reply:
Which one exactly?
The `hour` field will exceed uint8 temporarily during calculation but never does outside of a function.

Reply:
OK, if it's never happen outside of a function.